src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -215,34 +194,4 @@ export default ajaxCaller(class LaunchAnalysisModal extends Component {
       this.setState({ launchError: await error.text(), message: undefined })
     }
   }
-
```

Comment:
I'm sorry, @zarsky-broad . It pains me to remove code that you put good work into. I only do it because, with support for sets of sets, I don't think we'll be needing it anymore. :(

Reply:
The only part it should pain you to remove is my beautiful sweet progress bar that almost nobody got to see.